packages/@azure/servicebus/data-plane/package.json

```diff
@@ -28,7 +28,9 @@
     "long": "^4.0.0",
     "ms-rest-azure": "^2.5.9",
     "rhea-promise": "^0.1.13",
-    "tslib": "^1.9.3"
+    "tslib": "^1.9.3",
+    "@types/node": "^8.0.37",
```

Comment:
Should we add the need to have `@types/node` as a dependency in the guidelines?

Reply:
Will confirm tomorrow but I'm not sure it belongs here at all.

Reply:
Confirmed this does not belong here. At best peer dep, but it's possible we can remove the dep entirely. Investigating.